src/index.html

```diff
@@ -1,12 +1,46 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content">
+    <main class="calendar">
+      <div class="calendar__month calendar__month--start-day--sun calendar__month--month-length--29">
+        <div class="calendar__day calendar__day--1"></div>
```

Comment:
don't create separate class with date number for each day
use nth-child in css instead of the number in class name